src/components/Api.js

```diff
@@ -0,0 +1,139 @@
+export default class Api {
+  constructor(data) {
+    this._authorization = "0863e235-ec04-4229-a6bf-890245ffa3f4";
+  }
+
+  /*uploadInitialCards(data) {
+    console.log(data);
+    data.forEach((data) => {
+      this.uploadCard(data);
+    });
+  }*/
+
+  uploadCard({ name, link }) {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      method: "POST",
+      headers: {
+        authorization: this._authorization,
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        name: name,
+        link: link,
+      }),
+    }).then((res) =>
+      res.ok ? res.json() : Promise.reject(`Uh oh! Error: ${res.status}`)
+    );
+  }
+
+  fetchCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      method: "GET",
+      headers: {
+        authorization: this._authorization,
+        "Content-Type": "application/json",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)